Userland/Libraries/LibGUI/VimEditingEngine.h

```diff
@@ -51,6 +51,15 @@ class VimEditingEngine final : public EditingEngine {
 
     VimMode m_vim_mode { VimMode::Normal };
 
+    void vim_move_down(const KeyEvent&);
+    void vim_move_left(const KeyEvent&);
+    void vim_move_right(const KeyEvent&);
+    void vim_move_up(const KeyEvent&);
```

Comment:
Good note as well, I did that because move_down() etc would shadow/hide the move_down() that is inherited from EditingEngine, is there a precedent on doing this? :)